share/wake/lib/core/list.wake

```diff
@@ -259,9 +387,12 @@ export def take (length: Integer) (l: List a): List a =
 
 # drop: discard the first `num` elements
 #
+# Examples:
+#   ```
 #   drop 4 (seq 6) = 4, 5, Nil
 #   drop 6 (seq 6) = Nil
 #   drop 0 (seq 3) = 0, 1, 2, Nil
+#   ```
```

Comment:
I didn't even think about doing the same for `take`, but it might raise the same question.  Probably not a required example, though.

Reply:
OMG. A behaviour change based on sign? That's pure evil! What language does that?

Reply:
If other languages do this, I totally agree that both `take` and `drop` need an example with a negative number.

Reply:
I know Python does, and GNU head might be conceptually close enough to count (even though it has somewhat different semantics).  Looks like Javascript might as well.  I feel like I've come across a language before that actually does that *and* has a specific `take`/`drop` pair rather than using substring-style functions, but those examples are probably related enough to generalize to this.